common/main/en.xml

```diff
@@ -1376,7 +1376,6 @@ annotations.
 			<type key="collation" type="phonebook">Phonebook Sort Order</type>
 			<type key="collation" type="phonetic">Phonetic Sort Order</type>
 			<type key="collation" type="pinyin">Pinyin Sort Order</type>
-			<type key="collation" type="reformed">Reformed Sort Order</type>
```

Comment:
On one hand, it seems like we might want to keep display names until some time (maybe 1 year?) after their item starts being deprecated.
On the other hand, I suspect that this collation type and its display names are rarely used... so fine with me.
Is it normal for CLDR to remove display data immediately when deprecating something?

Reply:
CLDR stopped using the "reformed" type in CLDR 42. We decided in TC today that it was okay to remove the translations in this PR.